assets/stylesheets/_variables.scss

```diff
@@ -41,7 +41,7 @@ $shadow-modal: 0 3px 30px rgba($dark-gray-900, 0.2);
 
 // Editor Widths
 $sidebar-width: 280px;
-$content-width: 610px; // For the visual width, subtract 30px (2 * $block-padding + 2px borders). This comes to 580px, which is optimized for 70 characters.
+$content-width: 580px; // This is optimized for 70 characters.
```

Comment:
Heads up that this needs rebasing after https://github.com/WordPress/gutenberg/pull/17883 :-)